test-prow-e2e.sh

```diff
@@ -17,45 +27,61 @@ echo "Enabling Console Plugin for ODF Operator"
 oc patch console.v1.operator.openshift.io ${CONSOLE_CONFIG_NAME} --type=json -p="[{'op': 'add', 'path': '/spec/plugins', 'value':["${ODF_PLUGIN_NAME}"]}]"
 
 ODF_CONSOLE_IMAGE="$1"
-export ODF_CSV_NAME="$(oc get csv -n openshift-storage -o=jsonpath='{.items[?(@.spec.displayName=="OpenShift Data Foundation")].metadata.name}')"
+export ODF_CSV_NAME="$(oc get subscription -n openshift-storage -o=jsonpath='{.items[?(@.metadata.name=="odf-operator")].status.installedCSV}')"
 
 oc patch csv ${ODF_CSV_NAME} -n openshift-storage --type='json' -p \
   "[{'op': 'replace', 'path': '/spec/install/spec/deployments/1/spec/template/spec/containers/0/image', 'value': \"${ODF_CONSOLE_IMAGE}\"}]"
 
 # Installation occurs.
 
-# This is also the default case if the CSV is in "Installing" state initially.
-timeout 500 bash <<-'EOF'
-echo "waiting for ${ODF_CSV_NAME} clusterserviceversion to succeed"
-until [ "$(oc -n openshift-storage get csv -o=jsonpath="{.items[?(@.metadata.name==\"${ODF_CSV_NAME}\")].status.phase}")" == "Succeeded" ]; do
-  sleep 1
-done
-EOF
-
 ARTIFACT_DIR=${ARTIFACT_DIR:=/tmp/artifacts}
 SCREENSHOTS_DIR=gui-test-screenshots
-INSTALLER_DIR=${INSTALLER_DIR:=${ARTIFACT_DIR}/installer}
 
 function generateLogsAndCopyArtifacts {
-  oc get storageclusters ocs-storagecluster -n openshift-storage -o yaml > ${ARTIFACT_DIR}/storagecluster.yaml
-  oc get storagesystems -n openshift-storage -o yaml > ${ARTIFACT_DIR}/storagesystem_details.yaml
-  oc get nodes -o yaml > ${ARTIFACT_DIR}/node.yaml
-  oc get csvs -n openshift-storage > ${ARTIFACT_DIR}/csvs.yaml
+  oc cluster-info dump > ${ARTIFACT_DIR}/cluster_info.json
+  oc get catalogsource -n openshift-storage -o wide > ${ARTIFACT_DIR}/catalogsource.yaml
```

Comment:
```suggestion
  oc get catalogsources -A  > ${ARTIFACT_DIR}/catalogsource.yaml
```
can check all resources for this